resources/models.py

```diff
@@ -495,6 +495,13 @@ class Pier(AbstractAreaSection):
         verbose_name=_("price tier"),
         default=PriceTier.TIER_1,
     )
+    harbors_harbor = models.ForeignKey(
```

Comment:
Too bad the data doesn't allow to make this too an one-to-one mapping :-(

Reply:
Only a two harbors are mapped to several piers, otherwise it could be a 1:1
<img width="1156" alt="image" src="https://user-images.githubusercontent.com/15201480/111618560-302a2780-87ed-11eb-8272-0a23190c9f23.png">